.env.example

```diff
@@ -8,3 +8,6 @@ AWS_SECRET_ACCESS_KEY=
 STRIPE_PUBLIC_KEY=
 STRIPE_API_KEY=
 STRIPE_WEBHOOK_SECRET=
+
+# Frontend url
+FRONTEND_URL=http://localhost:3000
```

Comment:
How about completing that comment some more with:
```
Frontend URL, used for the Stripe success/cancel redirect
```

Reply:
okay